### PR TITLE
Addition of missing resource

### PR DESCRIPTION
**Pull request by joinimran**
URL was pointing to 404 page. Updated the link.

Problem: https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1735

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/tenant-wide-setup-for-increased-security.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ For secure environments, be sure to disable authentication for apps that do not
 
 In the meantime, use one of the following methods to accomplish this for SharePoint Online and OneDrive for Business:
 
-- Use PowerShell, see [Block apps that do not use modern authentication](https://docs.microsoft.com/intune-classic/deploy-use/block-apps-with-no-modern-authentication).
+- Use PowerShell, see [Block apps that do not use modern authentication](https://docs.microsoft.com/mem/intune/protect/app-modern-authentication-block).
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/tenant-wide-setup-for-increased-security.md]:
```suggestion
- Use PowerShell, see [Block apps that do not use modern authentication (ADAL)](https://docs.microsoft.com/mem/intune/protect/app-modern-authentication-block).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joinimran** [comment]
@mypil Ready to go live.

**Comment by denisebmsft** [comment]
Thank you @joinimran and @JohanFreelancer9

**Pull request merged**